packages/generate-design-tokens/css-generator.js

```diff
@@ -0,0 +1,357 @@
+        })
+        .join('\n');
+
+    return `
+${skinSelector} {
+    /* Palette colors */
+    ${paletteColors}
+
+    /* Raw palette colors, for use with rgba() */
+    ${rawPaletteColors}
+
+    /* Colors */
+    ${lightColors}
+
+    /* Border radius */
+    ${borderRadii}
+
+    /* Text */
+    ${textVars}
+}
+
+${skinSelector}${colorSchemeSelector('dark')} {
+    ${darkModeColors}
+}
+
+@media (prefers-color-scheme: dark) {
+    ${skinSelector} {
+        ${darkModeColors}
+    }
+    
+    ${skinSelector}${colorSchemeSelector('light')} {
+        ${forceLightModeColors}
+    }
+}
+
+${desktopMediaQuery} {
+    ${skinSelector} {
+        /* Text */
+        ${desktopTextVars}
+    }
+}
+`;
+};
+
+export const generateCommonCssSrc = (DESIGN_TOKENS_FOLDER) => {
+    const designTokensFile = fs.readFileSync(path.join(DESIGN_TOKENS_FOLDER, `movistar.json`), 'utf8');
+    const designTokens = JSON.parse(designTokensFile);
+
+    const borderRadiiClasses = Object.keys(designTokens.radius)
+        .map(
+            (radiusName) =>
+                `.${prefix}border-radius-${radiusName} {border-radius: var(${buildBorderRadiusVarName(
+                    radiusName
+                )})}`
+        )
+        .join('\n');
+
+    const textPresets = new Set([
+        'text1',
+        'text2',
+        'text3',
+        'text4',
+        'text5',
+        'text6',
+        'text7',
+        'text8',
+        'text9',
+        'text10',
+        ...Object.values(designTokens.text).flatMap(Object.keys),
+    ]);
+    const textClasses = Array.from(textPresets)
+        .map((presetName) => (presetName.startsWith('text') ? presetName.replace('text', '') : presetName))
+        .map(
+            (presetName) => `.${prefix}text-${presetName} {
+                font-size: var(${buildFontSizeVarName(presetName)});
+                line-height: var(${buildLineHeightVarName(presetName)});
+                font-weight: var(${buildFontWeightVarName(presetName)});
+            }`
+        )
+        .join('\n');
+
+    return `
+/* Default text color */
+[data-${prefix}skin] {
+    color: var(${buildColorVarName('textPrimary')});
+    background: var(${buildColorVarName('background')});
+}
+
+/* border-radius utility classes */
+${borderRadiiClasses}
+
+/* text utility classes */
+${textClasses}
+
+/* Responsive layout */
+.${prefix}responsive-layout {
+    padding-left: env(safe-area-inset-left);
+    padding-right: env(safe-area-inset-right);
+    margin: 0 var(--${prefix}responsive-layout-margin);
+    --${prefix}responsive-layout-margin: 16px;
+}
+
+${tabletMediaQuery} {
+    .${prefix}responsive-layout {
+        --${prefix}responsive-layout-margin: 24px;
+    }
+}
+
+${desktopMediaQuery} {
+    .${prefix}responsive-layout {
+        --${prefix}responsive-layout-margin: 40px;
+    }
+}
+
+${largeDesktopMediaQuery} {
+    .${prefix}responsive-layout {
+        --${prefix}responsive-layout-margin: calc((100vw - 1224px) / 2);
+    }
+}
```

Comment:
does it make sense to include some utility classes like `.mistica-responsive-layout` here?